src/vmshepherd/http/rpc_api.py

```diff
@@ -57,6 +57,31 @@ def rpc_logger(func):
         presets = await self.request.app.vmshepherd.preset_manager.list_presets()
         return list(presets.keys())
 
+    @enabled_checker
+    async def list_presets_with_tags(self):
+        """ Gets more verbose list of available presets
+
+        :return:  (dict of presets with tags)
+
+        :rtype: dict of dicts
+
+        Sample response:
+            ``{"preset1": {"tag1": "value1"}, "preset2": {"tag1": "value1"}}``
+        """
+        presets = await self.request.app.vmshepherd.preset_manager.list_presets()
+        return {name: preset.config['meta_tags'] for name, preset in presets.items()}
+
+    @enabled_checker
+    async def get_preset_tags(self, preset):
+        """ Get tags of preset
+
+        :arg string preset: preset name
+        :return: preset tags
+
+        :rtype: dict
```

Comment:
I would appreciate if that could be more specific. I assume that would be `Dict[str, str]`?

Reply:
IMHO it won't help, especially here `Dict[str,dict]`, without an example of such a structure is....

Reply:
Actualy here is a simple structure like {'k1': 'v1', 'k2': 'v2'}, so I'm agree with you it should be Dict[str, str], so in PR with type annotating fixies